src/main/java/org/fogbowcloud/arrebol/execution/docker/DockerCommandExecutor.java

```diff
@@ -0,0 +1,57 @@
+package org.fogbowcloud.arrebol.execution.docker;
+
+import org.apache.log4j.Logger;
+import org.fogbowcloud.arrebol.execution.docker.request.ExecInstanceResult;
+import org.fogbowcloud.arrebol.execution.docker.request.WorkerDockerRequestHelper;
+
+import static java.lang.Thread.sleep;
+
+public class DockerCommandExecutor {
+
+    private static final Logger LOGGER = Logger.getLogger(DockerCommandExecutor.class);
+    private final WorkerDockerRequestHelper workerDockerRequestHelper;
+
+    public DockerCommandExecutor(WorkerDockerRequestHelper workerDockerRequestHelper) {
+        this.workerDockerRequestHelper = workerDockerRequestHelper;
+    }
+
+    public Integer executeCommand(String command, String taskId) throws Exception {
+        LOGGER.info("Sending command to the [" + command + "] for the task [" + taskId + "]"
```

Comment:
I did no see much benefit on having two executeCommand method. It seems the public one does not do anything more than forwarding the request but logging

Reply:
.. I mean, I'd vote to move the code from the private method into the public one

Reply:
Done.